tests/countries/test_united_states.py

```diff
@@ -469,17 +513,95 @@ def test_columbus_day_states(self):
             "2022-10-10",
             "2023-10-09",
         )
-
-        for subdiv in ("AK", "AR", "DE", "FL", "HI", "NV"):
-            self.assertNoHoliday(self.state_hols[subdiv], dt)
-            self.assertNoHolidayName(name, self.state_hols[subdiv])
-
-        self.assertHolidayName("Native American Day", self.state_hols["SD"], dt)
+        for subdiv, change_year in (
+            ("AK", 2015),
+            ("DC", 2019),
+            ("ME", 2019),
+            ("NM", 2019),
+            ("NE", 2020),
+            ("VA", 2020),
```

Comment:
Not **Vermont (VT)**?
https://www.washingtonpost.com/nation/2019/04/20/vermont-passes-bill-abolishing-columbus-day-favor-indigenous-peoples-day/, first started in 2019 too, not 2020

Reply:
[Here](https://www.pewresearch.org/short-reads/2023/10/05/working-on-columbus-day-or-indigenous-peoples-day-it-depends-on-where-your-job-is/) says: 

> While the day is a legal holiday in Vermont, collective bargaining agreements allow state employees to use it as a floating holiday.

so I'm not sure if we should add this holiday for Vermont.